exporter/exporterhelper/common.go

```diff
@@ -121,9 +134,10 @@ func WithRetry(retrySettings RetrySettings) Option {
 
 // WithQueue overrides the default QueueSettings for an exporter.
 // The default QueueSettings is to disable queueing.
-func WithQueue(queueSettings QueueSettings) Option {
+// Permanent queue is not yet available for exporter helpers V2, they ignore StorageID field.
```

Comment:
I would at least raise a warning about this, possibly just error out

Reply:
The new exporter helper will get another option for the persistent queue with marshaller and unmarshaller as required params, and maybe another option for in-memory queue when the config doesn't have the `storage` param. So we won't be able to reuse this option anyway. I'll just put a panic here if it's used with the new exporter helper.